script.js

```diff
@@ -187,3 +187,17 @@ projectButtons.forEach((button) => {
     popupContainer.classList.remove('hidden');
   });
 });
+
+// Validation form
+const submit = document.querySelector('.button3');
+const errorMessage = document.querySelector('#error-message');
+submit.addEventListener('click', (event) => {
+  const emailInput = document.querySelector('#email');
+  const emailValue = emailInput.value;
+  if (emailValue !== emailValue.toLowerCase()) {
+    event.preventDefault();
+    errorMessage.innerHTML = alert('Please use only lowercase email addresses!');
+    return false;
+  }
```

Comment:
- [ ] I think it would be great if you display the error message beside the email field or submit button as mentioned in the requirements not using alert, so you can add an empty div and display the error message using js whenever the user write a capital letter in the email, I think it would be great if you make it this way so kindly fix it 💯😊🚀✅

 - Your way 
![image](https://github.com/MarkYegon7/Mark-Portfolio/assets/58553711/c32a1a02-4ece-4dd4-97e2-877d1d813166)


- you can make something like this instead
![image](https://github.com/MarkYegon7/Mark-Portfolio/assets/58553711/aa0cefeb-fc5b-409f-ae87-89b0a105e278)